examples/some-verifier/frontend/src/scss/layout/_btn.scss

```diff
@@ -0,0 +1,12 @@
+.btn-primary {
+  background-image: $button-gradient;
+  min-width: rfs-value(150px);
+}
+
+.btn-secondary {
+  @include button-variant($mineral-blue, $mineral-blue, $white);
+}
+
```

Comment:
It would be nice to close the dropDown similarly to how you can open the dropDown by clicking on its header. Currently, the only way to close it is to click on another dropDown header.

Reply:
Done!